pkg/util/iptables/testing/fake.go

```diff
@@ -26,80 +26,106 @@ import (
 )
 
 const (
+	// Destination represents the destination address flag
 	Destination = "-d "
-	Source      = "-s "
-	DPort       = "--dport "
-	Protocol    = "-p "
-	Jump        = "-j "
-	Reject      = "REJECT"
-	ToDest      = "--to-destination "
-	Recent      = "recent "
-	MatchSet    = "--match-set "
-	SrcType     = "--src-type "
-	Masquerade  = "MASQUERADE "
+	// Source represents the source address flag
+	Source = "-s "
+	// DPort represents the destination port
+	DPort = "--dport "
+	// Protocol represents the protocol flag which takes input by number of name
+	Protocol = "-p "
+	// Jump represents jump flag specifies the jump target
+	Jump = "-j "
+	// Reject specifies the reject target
+	Reject = "REJECT"
+	// ToDest represents the --to-destination flag used to specify the destination address in DNAT
+	ToDest = "--to-destination "
+	// Recent represents the sub-command recent that allows to dynamically create list of IP address to match against
+	Recent = "recent "
+	// MatchSet represents the --match-set flag which match packets against the specified set
+	MatchSet = "--match-set "
+	// SrcType represents the --src-type flag which matches if the source address is of given type
+	SrcType = "--src-type "
+	// Masquerade represents the target that is used in nat table.
+	Masquerade = "MASQUERADE "
 )
 
+// Rule holds a map of rules.
 type Rule map[string]string
 
-// no-op implementation of iptables Interface
+// FakeIPTables no-op implementation of iptables Interface.
```

Comment:
`FakeIPTables is a no-op...`
(add "is")

Reply:
sorry for the grammar mistake 😉 , changed it.